components/datadog/agent/host_windowsos.go

```diff
@@ -80,8 +80,24 @@ func (am *agentWindowsManager) getAgentConfigFolder() string {
 func (am *agentWindowsManager) restartAgentServices(transform command.Transformer, opts ...pulumi.ResourceOption) (*remote.Command, error) {
 	// TODO: When we introduce Namer in components, we should use it here.
 	cmdName := am.host.Name() + "-" + "restart-agent"
+	cmd := fmt.Sprintf(`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As discussed I would rather have a more robust agent start.

Noticed we don't follow the public documentation, we should directly call the binary https://docs.datadoghq.com/agent/basic_agent_usage/windows/?tab=commandline#agent-commands

Reply:
Yes ideally we'd want the subcommand to handle this case. But I think this is an issue known for a long time and I do not think Windows team planned to work on it yet. So I think this is the only option we have to stabilize tests.

Sounds good to modify the script to call the binary directly instead of wrapping it in `Start-Process`